tests/test_rebase_command.rs

```diff
@@ -426,6 +426,75 @@ fn test_rebase_with_descendants() {
     o  a
     o
     "###);
+    test_env.jj_cmd_success(&repo_path, &["undo"]);
+    // Simple case
+    let stdout = test_env.jj_cmd_success(&repo_path, &["rebase", "-s=c", "-s=d", "-d=a"]);
+    insta::assert_snapshot!(stdout, @r###"
+    Rebased 2 commits
+    Working copy now at: 92c2bc9a8623 d
+    Added 0 files, modified 0 files, removed 2 files
+    "###);
+    insta::assert_snapshot!(get_log_output(&test_env, &repo_path), @r###"
+    o  c
+    │ @  d
+    ├─╯
+    │ o  b
+    o │  a
+    ├─╯
+    o
+    "###);
+
+    test_env.jj_cmd_success(&repo_path, &["undo"]);
+    // Reminder of the setup
+    insta::assert_snapshot!(get_log_output(&test_env, &repo_path), @r###"
+    @  d
+    o    c
+    ├─╮
+    o │  b
+    │ o  a
+    ├─╯
+    o
+    "###);
+    let stdout = test_env.jj_cmd_success(&repo_path, &["rebase", "-s=b", "-s=d", "-d=a"]);
+    insta::assert_snapshot!(stdout, @r###"
+    Rebased 3 commits
+    Working copy now at: f1e71cb78a06 d
+    Added 0 files, modified 0 files, removed 2 files
+    "###);
+    insta::assert_snapshot!(get_log_output(&test_env, &repo_path), @r###"
+    o  c
+    o  b
+    │ @  d
+    ├─╯
+    o  a
+    o
+    "###);
+    test_env.jj_cmd_success(&repo_path, &["undo"]);
```

Comment:
Same here: could you add a blank line and a description saying what this is testing?

Reply:
Done.